### PR TITLE
[FSPE-6494] - Added e2e test for chi accordion component

### PR DESCRIPTION
**Pull request by shintuviswambharan**
Added e2e test for chi accordion component
@SwathiLingareddy / @janibolkvadze  could you please review

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Build was ABORTED. See http://jenkins.elasticbox.com/job/chi_pull_request/1554/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://5d4d741392df72f0ab106f8175c150f0fb7b7dc9-chi-pr.enterprise.ctl.io" target="_blank"> https://5d4d741392df72f0ab106f8175c150f0fb7b7dc9-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1555/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://50bb8002339d670384b94330ad68144c84169577-chi-pr.enterprise.ctl.io" target="_blank"> https://50bb8002339d670384b94330ad68144c84169577-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1556/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://f8e030f2d99d0a13a7534179f28026c1a5fa1989-chi-pr.enterprise.ctl.io" target="_blank"> https://f8e030f2d99d0a13a7534179f28026c1a5fa1989-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1560/ for more details.

**Review by SwathiLingareddy — APPROVED** [review]
Changes looks fine to me

File: test/chi/js/accordion.pug [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
+title: Accordion
+---
+script(src='/js/chi.js')
```
Comment (janibolkvadze) [inline test/chi/js/accordion.pug:4]:
```suggestion
script(src='/js/chi.js')

```

File: test/chi/js/accordion.pug [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      .chi-accordion__title Accordion 1
+      i.chi-icon.icon-chevron-down
+    .chi-accordion__content
+      p.chi-accordion__text Content goes here
+  .chi-accordion__item.-expanded(data-cy='item-2')
+    button.chi-accordion__trigger
+      .chi-accordion__title Accordion 2
+      i.chi-icon.icon-chevron-down
+    .chi-accordion__content
+      p.chi-accordion__text Content goes here
```
Comment (janibolkvadze) [inline test/chi/js/accordion.pug:17]:
```suggestion
      p.chi-accordion__text Content goes here

```

File: cypress/integration/chi-js/accordion.spec.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+          .click({ multiple: true });
+        cy.get('@firstItem')
+          .should('have.not.class', '-expanded')
+          .find('.chi-accordion__content')
+          .should('have.not.css', 'display', 'block');
+      });
+    });
+  });
+
+  describe('Nested accorion should expand and collapse accordingly with nested items', () => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/accordion.spec.js:35]:
```suggestion
  describe('Nested accordion should expand and collapse accordingly with nested items', () => {
```

File: cypress/integration/chi-js/accordion.spec.js [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+    it('Panel should expand  when main item clicked', () => {
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/accordion.spec.js:36]:
```suggestion
    it('Panel should expand  when trigger is clicked', () => {
```

File: cypress/integration/chi-js/accordion.spec.js [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+        cy.get('.chi-accordion__item').should('have.not.class', '-expanded');
+      });
+    });
+  });
+
+  describe('Accordion should appear inactive in disabled state', () => {
+    it('Accordion should appear inactive in disabled state', () => {
+      cy.get('#disabled-accordion')
+        .find('.chi-accordion__trigger')
+        .should('have.class', '-disabled');
```
Comment (janibolkvadze) [inline cypress/integration/chi-js/accordion.spec.js:135]:
```suggestion
        .should('have.class', '-disabled')
        .should('have.css', 'pointer-events', 'none');
```

File: test/chi/js/accordion.pug [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+                button.chi-accordion__trigger
+                  .chi-accordion__title Accordion 2.1
+                  i.chi-icon.icon-chevron-down
+                .chi-accordion__content
+                  p.chi-accordion__text Content goes here
+button#toggle-accordion-1.chi-button Toggle item 1
+button#toggle-accordion-2.chi-button Toggle item 2
+button#show-accordion.chi-button Show item 3
+button#hide-accordion.chi-button Hide item 3
+#individual-accordion-items.chi-accordion.-portal.-w--75
```
Comment (janibolkvadze) [inline test/chi/js/accordion.pug:69]:
```suggestion

button#toggle-accordion-1.chi-button Toggle item 1
button#toggle-accordion-2.chi-button Toggle item 2
button#show-accordion.chi-button Show item 3
button#hide-accordion.chi-button Hide item 3

#individual-accordion-items.chi-accordion.-portal.-w--75
```

File: test/chi/js/accordion.pug [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+            p.chi-accordion__text Content of Accordion item 2
+        #invividual-accordion-item-3.chi-accordion__item
+          button.chi-accordion__trigger
+            .chi-accordion__title Accordion item 3
+            i.chi-icon.icon-chevron-down
+          .chi-accordion__content
+            p.chi-accordion__text Content of Accordion item 3
+button#expand-all.chi-button Expand all
+button#collapse-all.chi-button Collapse all
+#expand-collapse.chi-accordion.-portal.-w--75
```
Comment (janibolkvadze) [inline test/chi/js/accordion.pug:96]:
```suggestion

button#expand-all.chi-button Expand all
button#collapse-all.chi-button Collapse all

#expand-collapse.chi-accordion.-portal.-w--75
```

File: test/chi/js/accordion.pug [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+                  i.chi-icon.icon-chevron-down
+                .chi-accordion__content
+                  p.chi-accordion__text Content of Accordion item 2.1
+              .chi-accordion__item
+                button.chi-accordion__trigger
+                  .chi-accordion__title Accordion item 2.2
+                  i.chi-icon.icon-chevron-down
+                .chi-accordion__content
+                  p.chi-accordion__text Content of Accordion item 2.2
+#disabled-accordion.chi-accordion.-portal.-w--75
```
Comment (janibolkvadze) [inline test/chi/js/accordion.pug:141]:
```suggestion

#disabled-accordion.chi-accordion.-portal.-w--75
```

File: test/chi/js/accordion.pug [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+  .chi-accordion__item
+    button.chi-accordion__trigger.-disabled
+      i.chi-icon.icon-chevron-down
+      .chi-accordion__title Accordion 1
+    .chi-accordion__content
+      p.chi-accordion__text Content goes here
+script.
```
Comment (janibolkvadze) [inline test/chi/js/accordion.pug:148]:
```suggestion

script.
```

**Comment by janibolkvadze** [comment]
@shintuviswambharan Awesome! Thank you!

**Review by janibolkvadze — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://b954319b1b47ed194eb88d0dd33660b28069d256-chi-pr.enterprise.ctl.io" target="_blank"> https://b954319b1b47ed194eb88d0dd33660b28069d256-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1567/ for more details.

**Comment by janibolkvadze** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://b954319b1b47ed194eb88d0dd33660b28069d256-chi-pr.enterprise.ctl.io" target="_blank"> https://b954319b1b47ed194eb88d0dd33660b28069d256-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1568/ for more details.

**Comment by jllr** [comment]
@SwathiLingareddy can you please review this PR in order to merge it? Thanks in advance

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-nna97w, https://ops.elasticbox.com/services/instances/i-yvd0bd, https://ops.elasticbox.com/services/instances/i-spxfr7, https://ops.elasticbox.com/services/instances/i-0brjrg

**Pull request merged**